src/components/sides/TaskInfo.vue

```diff
@@ -1315,7 +1315,7 @@ export default {
         this.loading.setFrameThumbnail = true
         let frame = null
         if (isUseCurrentFrame) {
-          frame = this.currentFrameRaw
+          frame = this.currentFrameRaw + 1
```

Comment:
Please add a comment on this line to explain why "+ 1".
It will be easier to read and understand the code later (git blame is not viable over time)